net/vpnbypass/files/vpnbypass.init

```diff
@@ -1,9 +1,9 @@
 #!/bin/sh /etc/rc.common
 
-START=92
+START=99
```

Comment:
With priority 94 it was still starting too early to not be able to obtain a WAN IP address despite the boot() being what it is.
With priority 99 it starts fine.